src/script/components/Modals/GroupCreation/GroupCreationModal.tsx

```diff
@@ -17,11 +17,12 @@
  *
  */
 
-import React, {useEffect, useState} from 'react';
+import React, {useEffect, useState, useMemo} from 'react';
 import {container} from 'tsyringe';
 import cx from 'classnames';
 import {amplify} from 'amplify';
 import {WebAppEvents} from '@wireapp/webapp-events';
+import {Select, StyledApp} from '@wireapp/react-ui-kit';
```

Comment:
```suggestion
import {Select, StyledApp, THEME_ID} from '@wireapp/react-ui-kit';
```
The change to the ui-kit is live on dev